development/env

```diff
@@ -0,0 +1,20 @@
+MYSQL_HOST=db
+MYSQL_ROOT_PASSWORD=myrootpassword
+MYSQL_USER=magento
+MYSQL_PASSWORD=magento
```

Comment:
Let's use the username and password we agreed for development docker services:
```
user: metrilo
password: metrilo1
```
I'm changing the wordpress ones currently.

Reply:
👍

Reply:
Remind, remind.